features/step_definitions/user_sign_in_steps.rb

```diff
@@ -0,0 +1,14 @@
+Given(/^I click on "(.*?)"$/) do |arg1|
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `arg1`. You can omit the argument if you don't care about it.